<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>api_tool</class>
 <widget class="QFrame" name="api_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>API Tool</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>951</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="horizontalContainer">
       <layout class="QHBoxLayout" name="MainContent">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>GET</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>POST</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PUT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DELETE</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>29</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetContainer">
       <layout class="QVBoxLayout" name="tabWidgetLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="font">
           <font>
            <family>.AppleSystemUIFont</family>
           </font>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_1">
           <attribute name="title">
            <string>Parameters</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_authentication">
           <attribute name="title">
            <string>Authentication</string>
           </attribute>
           <layout class="QFormLayout" name="authLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="usernameLabel">
              <property name="text">
               <string>Username:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="usernameEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="text">
               <string>Password:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordEdit">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_body">
           <attribute name="title">
            <string>Body</string>
           </attribute>
           <layout class="QVBoxLayout" name="bodyLayout">
            <item>
             <widget class="QTextEdit" name="bodyTextEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListView" name="listView"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
